src/TreeView/TreeView.tsx

```diff
@@ -86,7 +86,7 @@ const UlBox = styled.ul<SxProp>`
   .PRIVATE_TreeView-item {
     outline: none;
 
-    &:focus-visible > div {
+    &.focus-visible > div {
```

Comment:
this is to fix focus not showing in safari

Reply:
Could we do both?
```suggestion
    &:focus-visible > div,
    &.focus-visible > div  {
```